### PR TITLE
Add cloud identity SSO guide

### PR DESCRIPTION
**Pull request by harrietgrace**
A copy of the other guides, but for Cloud Identity. It's not a complex set up, so this draft has the majority of it in the steps. Just a few bits to clean up, and to decide if we want to add this as a guide 🤷🏻‍♀️

### CODE REVIEW

**Comment by keithpitt** [comment]
I've been seeing usage of this service up, so I think a guide would be a good thing! Also helps when people google for "cloud identity buildkite" and want to know if we integrate.

File: pages/integrations/sso/cloud_identity.md.erb [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+</div>
+
+Log into your Google Admin Console, and follow these steps:
+
+1. In the 'Apps Settings' area of the console, select 'SAML Apps'
+1. Click the + button to open the Enable SSO for SAML Application modal 
+1. Choose 'Setup my own custom app'
+1. Copy down the SSO URL and Entity ID, and download the Certificate 
+1. Give your application a name, for example 'Buildkite'
+1. ACS URL = the URL provided by Buildkite support, Entity ID = https://buildkite.com
```
Comment (toolmantim) [inline pages/integrations/sso/cloud_identity.md.erb]:
```suggestion
1. For 'ACS URL' use the URL unique SSO service URL provided by support
1. For `Entity ID` use `https://buildkite.com`
```

Comment (harrietgrace) [reply pages/integrations/sso/cloud_identity.md.erb]: This line is why this PR is a draft 😂

Comment (toolmantim) [reply pages/integrations/sso/cloud_identity.md.erb]: Oh sorry, I didn’t see that! 😬🙃

**Comment by harrietgrace** [comment]
@toolmantim I've added in the last bits for this. Do you think the spacing is too close in this list? It feels a bit cramped to me. 

image

**Comment by harrietgrace** [comment]
Yeah sounds good - do you want me to add it as a to-do anywhere?

**Comment by toolmantim** [comment]
Yeah, maybe an issue here on the docs repo (with a link and that screenshot?), now that it's going to all live in here?

**Pull request merged**